test/functional/mempool_rebroadcast.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python3
```

Comment:
In _[mempool] mempool tracks wallet txns & ensures successful initial broadcast_, there are a few minor changes to this file (eg whitespace, comment corrections) that I think should just be merged into the first commit.